stubs/click-spinner/click_spinner/_version.pyi

```diff
@@ -1,5 +1,5 @@
-from typing import Any
+from typing import Literal
 
 version_json: str
 
-def get_versions() -> dict[str, Any]: ...
+def get_versions() -> dict[Literal["dirty", "error", "full-revisionid", "version"], str | bool | None]: ...
```

Comment:
I would consider this kind of annotation a regression. This requires `isinstance` checks, and typing the possible keys as literal has little practical use. That said, this would be a perfect candidate for a `TypedDict`.